interface/super/edit_list.php

```diff
@@ -1419,6 +1418,35 @@ function lister() {
             for ($i = 0; $i < 3; ++$i) {
                 writeITLine(array());
             }
+        } elseif ($list_id == 'external_patient_education') {
+            $sql = "SELECT lo.* FROM list_options AS lo
+                    RIGHT JOIN list_options lo2
+                        ON lo2.option_id = lo.list_id
+                        AND lo2.list_id = 'lists'
+                        AND lo2.edit_options = 1
+                    WHERE lo.list_id = ?
+                        AND lo.edit_options = 1
+                    ORDER BY seq, title {$sql_limits}";
+            $query = sqlStatement($sql, [$list_id]);
+            $results = [];
+            global $opt_line_no, $list_id;
+
+            while ($row = sqlFetchArray($query)) {
+                $row['line_no'] = $opt_line_no;
+                $results[] = $row;
+                ++$opt_line_no;
+            }
+
+            for ($i = 0; $i < 3; ++$i) {
+                $results[] = ['line_no' => $opt_line_no];
+                ++$opt_line_no;
+            }
+
+            $vars = [
+                'rows' => $results,
+            ];
+            echo $twig->render('super/lists/external_patient_education/partials/row.html.twig', $vars);
```

Comment:
shouldn't need this block of code. since all doing is changing Notes label to URL label, can just leverage code that is already being used.